scripts/fuzz_shell.js

```diff
@@ -134,6 +134,29 @@ function logValue(x, y) {
   console.log('[LoggingExternalInterface logging ' + printed(x, y) + ']');
 }
 
+// Table get/set operations need a BigInt if the table has 64-bit indexes. This
+// adds a proper cast as needed.
+function castTableIndex(index) {
```

Comment:
How about `toAddressType(table, index)`?  (Passing table as arg0 would allow it to work on any table)